assets/css/components/_news-and-views.scss

```diff
@@ -0,0 +1,40 @@
+figure {
+  height: rem(298px);
+  width: rem(311px);
```

Comment:
It may not be a good idea to set `width` to a specific size especially on the mobile device. The grid for each article is stretching outside of the screen on mobile:

<img width="464" alt="Screen Shot 2020-03-24 at 4 55 23 PM" src="https://user-images.githubusercontent.com/572877/77476091-4aa30e00-6df0-11ea-8d97-f5304e82dbeb.png">

Reply:
Pages on medium sized screen also don't look right. Removing the fixed width on `figure` might help to fix it too.

The grid at the right stretches out of the screen on Ipad portrait and landscape views:
<img width="800" alt="Screen Shot 2020-03-24 at 4 56 05 PM" src="https://user-images.githubusercontent.com/572877/77476299-b4231c80-6df0-11ea-83c6-7b0749563b09.png">
<img width="1016" alt="Screen Shot 2020-03-24 at 4 56 14 PM" src="https://user-images.githubusercontent.com/572877/77476311-b84f3a00-6df0-11ea-9275-16018f62951d.png">

Reply:
In terms of the width for each grid item, I think using percentage would be more flexible.